src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -795,6 +795,9 @@ Copyright (c) .NET Foundation. All rights reserved.
       <ResolvedFileToPublish Include="@(_RazorDebugSymbolsIntermediatePath)" Condition="'$(CopyOutputSymbolsToPublishDirectory)'=='true'">
         <RelativePath>@(_RazorDebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
+
+      <!-- Pass assembly to linker -->
+      <TrimmerAdditionalAssembly Include="@(RazorIntermediateAssembly)" Condition="'$(CopyBuildOutputToPublishDirectory)'=='true'">
```

Comment:
```suggestion
      <TrimmerAdditionalAssembly Include="@(RazorIntermediateAssembly)" Condition="'$(CopyBuildOutputToPublishDirectory)'=='true'" />
```

Reply:
Did you just self-suggest on your own PR? That's amazing. 

![image](https://user-images.githubusercontent.com/1430011/63070380-d2768080-becf-11e9-9f17-7c1a5e8dbc5e.png)